ZeroWasteFriendlyTests/ActivityTiles/ActivityTilesCreatorTests.swift

```diff
@@ -0,0 +1,35 @@
+//
+//  ActivityTilesCreatorTests.swift
+//  ZeroWasteFriendly
+//
+//  Created by Marek Skrzelowski on 05/02/2020.
+//  Copyright © 2020 vandermesis. All rights reserved.
+//
+
+import Quick
```

Comment:
I am now a fan of getting rid of as much libraries as possible, have you been thinking about using native testing?
just a thought, If you'd like to try it then It might be a good idea as you're already acknowledged with Quick and Nimble.